src/sonic-ctrmgrd/ctrmgr/container

```diff
@@ -11,7 +11,7 @@ import datetime
 import docker
 from swsscommon import swsscommon
 
-CTR_STATE_SCR_PATH = '/usr/share/sonic/scripts/container_startup.py'
+CTR_STATE_SCR_PATH = '/etc/sonic/remote_ctr.config.json'
```

Comment:
>remote_ctr.config.json [](http://example.com/codeflow?start=33&length=22)

Original it is a script, and name SCR is script. Why it is now a json file?

Reply:
The logic here is to check whether k8s is enabled, if the file exists, it means enabled. So just check file whether exists, don't care the content.

Reply:
Do you also want to change CTR_STATE_SCR_PATH -> CTRMGRD_SERVICE_PATH

Reply:
Thanks, changed.